chat-client-ui-types/src/uiContracts.ts

```diff
@@ -117,3 +117,8 @@ export interface CopyCodeToClipboardMessage {
     command: typeof COPY_TO_CLIPBOARD
     params: CopyCodeToClipboardParams
 }
+
+export interface ErrorResult {
+    message: string
+    type: 'InvalidRequest' | 'InternalError' | 'UnknownError' | string
+}
```

Comment:
Looks interesting, is this intended for general use, or only opentab?

Would it make sense for this be modeled as `Result` (and include 'Success')), instead of `ErrorResult`?

Reply:
It can be for general use, but handling responses through `postMessage` would require improvement in general.